style.css

```diff
@@ -788,6 +885,33 @@ body {
   .menu-section-active {
     width: 0;
   }
+
+  .popupBody {
```

Comment:
Kindly stick to the Figma template as the previous reviewer requested and fix the overflowing issue and add some space above buttons.
![Screen Recording 2023-02-11 at 11 06 54 PM](https://user-images.githubusercontent.com/25887253/218279217-dc833176-7a51-4fcd-9829-fa1b93bc3f89.gif)